python/ppml/example/fgboost_regression/fgboost-tutorial.md

```diff
@@ -0,0 +1,172 @@
+# Tutorial for Vertically Federated XGBoost
+This example provides a step-by-step tutorial of running a FGBoost Regression task with 2 parties.
+## 1. Key Concepts
+### 1.1 FGBoost 
+**FGBoost** stands for Federated Gradient Boosted Tree algorithm. It allows multiple parties to run a federated gradient boosted decision tree application.
+### 1.2 PSI
+**PSI** stands for Private Set Intersection algorithm. It compute the intersection of data from different parties and return the result so that parties know which data they could collaborate with.
+### 1.3 SGX
+**SGX** stands for [Software Guard Extensions](https://www.intel.com/content/www/us/en/developer/tools/software-guard-extensions/overview.html). It helps protect data in use via unique application isolation technology. The data of application running in SGX would be protected.
+
+### 1.4 FL Server and Client
```

Comment:
Focus on:
(1) FL Server and Client are part of BigDL PPML
(2) A centralized FL Server runs in SGX, aggregates intermediate results of clients, and return aggregated results (loss, gradients)
(3) An FL Client runs on each party, computes intermediate result on local data, and updates partial model based on aggregated results from the FL Server